docs/reverse_proxy.md

```diff
@@ -111,9 +117,13 @@ example.com:8448 {
     ProxyPassReverse /_synapse/client http://127.0.0.1:8008/_synapse/client
 </VirtualHost>
 
+Listen 8448
```

Comment:
Just for my own curiosity, this is indeed required as per https://httpd.apache.org/docs/2.4/bind.html.

> To implement a VirtualHost, the server must first be told to listen to the address and port to be used. Then a <VirtualHost> section should be created for the specified address and port to set the behavior of this virtual host. Note that if the <VirtualHost> is set for an address and port that the server is not listening to, it cannot be accessed.